app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    class TargetsController < ApplicationController
+      def create
+        @target = Target.new(target_params)
+        if @target.save
+          render :show, status: :created
+        else
+          render json: @target.errors, status: :unprocessable_entity
+        end
```

Comment:
Maybe you can do `Target.create!(... ` and let the `ApplicationController` to handle errors.